datahub-web-react/src/app/ingest/source/IngestionSourceTableColumns.tsx

```diff
@@ -63,7 +64,8 @@ const CliBadge = styled.span`
 `;
 
 export function TypeColumn(type: string, record: any) {
-    const iconUrl = sourceTypeToIconUrl(type);
+    let iconUrl = useGetSourceLogoUrl(record.platformUrn || '');
```

Comment:
hmm--- this seems overly complex. why aren't we just fetching source.platform.logoUrl in the initial data pull?

Reply:
ugh i know. so it turns out that I don't believe that these ingestion sources often actually have a platform associated with them, so it's usually null (idk how the platform gets set on these sources), so yes I agree it would make sense to just get the logoUrl. The only thing is that here I'm trying to short-circuit and use the logo we have in memory in the react bundle before we fetch to get the image from where we host it on github somewhere.

So really this code block is almost always (if not always always?) going to return null and go to the next line to get the icon purely based on what we have in react.